agnoster.zsh-theme

```diff
@@ -122,10 +122,20 @@ prompt_status() {
 
 # Display current virtual environment
 prompt_virtualenv() {
+  local env='';
+
   if [[ -n $VIRTUAL_ENV ]]; then
+    env=$VIRTUAL_ENV
+  fi
+
+  if [[ -n $CONDA_DEFAULT_ENV ]]; then
```

Comment:
@MichaelAquilina I assume you're right. As this is not my PR I cannot say for sure. But it makes sense.